Homework_3/task3/forms.py

```diff
@@ -0,0 +1,10 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField
+from wtforms.validators import DataRequired, Email, EqualTo, Length
+
+
+class LoginForm(FlaskForm):
+    username = StringField('Username')
+    email = StringField('Email', validators=[DataRequired(), Email()])
```

Comment:
Тут можно использовать `EmailField`
```suggestion
    email = EmailField('Email', validators=[DataRequired(), Email()])
```